src/common/util/submit-data.js

```diff
@@ -11,7 +11,7 @@ export const submitData = {}
  * @param {string} jsonp
  * @returns {Element}
  */
-submitData.jsonp = function jsonp (url, jsonp) {
+submitData.jsonp = function jsonp ({ url, jsonp }) {
```

Comment:
Need to update the jsdocs.

Reply:
we should discuss, because i'm in the opposite camp.  I think named args, esp. when theres like 6-7 of them, is extremely messy and makes passing optional arguments a huge pain in comp. with object args.

For instance, the send* calls all can have many arguments but only 1 is required.

Reply:
StandardJS doesn't have any opinion on this that I can find. I prefer using an options object because it makes adding parameters easier and it's easier to tell what the parameters are for when reading a method invocation without having to go lookup that method in the code. However, it may add overhead in memory usage, garbage collection, and larger bundles.